be/tool/beck.c

```diff
@@ -1435,8 +1451,12 @@ static void emap_act(struct action *act, struct m0_be_tx *tx)
 
 		rc = emap_entry_lookup(adom, emap_key->ek_prefix, 0, &it);
 		/* No emap entry found for current stob, insert hole */
+		fid.f_container = emap_key->ek_prefix.u_hi;
```

Comment:
Please check if rather than passing structure m0_fid, passing m0_uint128 will reduce the code. 
Typecasting if ok, will also reduce the code